SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you believe you've identified a security vulnerability in BigBone (a bug that allows something to happen that shouldn't be possible), please
+[open a BigBone security issue](https://github.com/andregasser/bigbone/security/advisories/new).
+
+You should not report such issues on public GitHub issues or in other public spaces to give us time to publish a fix for the issue without exposing BigBone's users to increased risk.
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
```

Comment:
This sounds like an internal comment for writing this file rather than something that we should present to library users. I suggest rephrasing to something like

> The following versions of BigBone are supported with security updates:

if we even want to offer this support for more than one major version, or simply

> Only the latest major version of BigBone will receive security updates.

if we don't.

Reply:
Ah crap, sure will fix this today. Agreed.

Reply:
Fixed.